src/View/Shortcodes/EmbedShortcodeProvider.php

```diff
@@ -16,6 +16,7 @@
 use SilverStripe\View\Parsers\ShortcodeHandler;
 use SilverStripe\View\Parsers\ShortcodeParser;
 use SilverStripe\Control\Director;
+use SilverStripe\Core\Config\Configurable;
```

Comment:
There was a use `use SilverStripe\Dev\Deprecation;` statement here that was causing a conflict.